File: src/main/java/com/honghailt/cjtj/service/dto/CrowdList.java
package com.honghailt.cjtj.service.dto;

public class CrowdList {

    private Long price; // 出价
    private String crowdName; //人群名称
    private String crowdDesc; // 人群描述
    private String optionName;  // 选项名称
    private DirectionalLabel directionalLabel;


    public Long getPrice() {
        return price;
    }

    public void setPrice(Long price) {
        this.price = price;
    }

    public String getCrowdName() {
        return crowdName;
    }

    public void setCrowdName(String crowdName) {
        this.crowdName = crowdName;
    }

    public String getCrowdDesc() {
        return crowdDesc;
    }

    public void setCrowdDesc(String crowdDesc) {
        this.crowdDesc = crowdDesc;
    }

    public String getOptionName() {
        return optionName;
    }

    public void setOptionName(String optionName) {
        this.optionName = optionName;
    }

    public DirectionalLabel getDirectionalLabel() {
        return directionalLabel;
    }

    public void setDirectionalLabel(DirectionalLabel directionalLabel) {
        this.directionalLabel = directionalLabel;
    }
}

File: src/main/webapp/app/group/add/CrowdList.ts

export class CrowdList {
    public price?: number; // 出价
    public crowdName?: string; //人群名称
    public crowdDesc?: string; // 人群描述
    public optionName?: string;  // 选项名称
    public directionalLabel?: DirectionalLabel;
}

export class DirectionalLabel {
    public labelId?: number;  // 标签id
    public targetId?: number; // 定向id
    public targetType?: string; // 定向类型
    public labelValue?: string; // 标签值
    public options?: Array<Options> = new Array<Options>();
}

export class Options {
    public optionName?: string;  // 选项名称
    public optionValue?: string; // 选项值
    public optionDesc?: string; // 选项描述
    public checked?: boolean; // 是否选中
}
